integration_test/integration-test.go

```diff
@@ -0,0 +1,313 @@
+package main
```

Comment:
you can run :
`gofmt -s -w integration-test.go` to format following some go standard.

Reply:
Reformatted in this [commit](https://github.com/dengyijia/WAF-wasm/pull/3/commits/f89b6330fe8f9137369359067fe79d8c7816e79b)